src/components/HomePage.jsx

```diff
@@ -3,9 +3,11 @@ import { Grid } from "semantic-ui-react";
 import { Article } from "../modules/apiHelper";
 import ArticleItem from "./ArticleItem";
 import { useSelector } from "react-redux";
+import DisplayedArticle from "./DisplayedArticle";
 
 const HomePage = () => {
   const { articles } = useSelector((state) => state);
+  const { article } = useSelector((state) => state);
```

Comment:
`const { article } = useSelector((state) => state);`  I feel this could be further refactored
`const { article, articles } = useSelector((state) => state);`

Reply:
Good thinking